docs/1.0. Architecture.md

```diff
@@ -32,9 +32,9 @@ IPMX (Internet Protocol Media Experience) is a proposed set of open standards an
 
 An application which controls IPMX-compatible NMOS Nodes: discovers them, makes connections etc. An analogue of Broadcast Controller for Pro AV use cases.
 
-**IPMX Controller**
+**IPMX Device Controller**
 
-An IS-04 Device Control. It provides an HTTP RESTful API (IS-XX) which provides information about and control of an IPMX-driven NMOS Device.
+A software for setting up an IPMX-compatible device and synchronizing its state with NMOS APIs. It has an HTTP RESTful API called IPMX Device Control API which provides information about and control of an IPMX-driven device.
```

Comment:
Having had some internal discussions, I struggled to describe this... 

Thoughts?

```suggestion
A mechanism for configuring an IPMX-compatible device and transferring its state. It has an API called "IPMX Device Control" which provides information and control of the device. This conceptual component is the link to the physical EDID medium.
```

Reply:
It looks better than mine but I'm not sure about two points.
1. What does `transferring` mean? Shouldn't we explicitly mention that IPMX Device Controller transfers device state to NMOS APIs?
2. I guess, the last sentence isn't fully correct. IPMX Device Controller may be expanded with functionality not related to EDID. Another reason to avoid EDID mentioning here, as for me, is relative generalism of the proposed approach which allows non-HDMI devices to perform the same metadata exchange.

Reply:
1. REST == Representational state **transfer**, I was trying to use the verb over the acronym 
2. I guess that's a topic we should clear up, the work in under _edid connection managent_.. My concern is I am not sure how much father we could expand the topics